draft-ietf-quic-transport.md

```diff
@@ -666,64 +664,228 @@ Version negotiation uses unprotected data.  The result of the negotiation MUST
+which uses TLS.
+
+QUIC provides this stream with reliable, ordered delivery of data.  In return,
+TLS provides QUIC with:
+
+* authenticated key exchange, always for the server, optionally for the client
+
+* keying material for use in packet packet protection
 
-### Transport Parameters and Options
+* authenticated values for the transport parameters of the peer (see
+  {{transport-parameters}})
 
-During connection establishment, the handshake must negotiate various transport
-parameters.  The currently defined transport parameters are described later in
-the document.
+* authenticated confirmation of version negotiation (see {{version-validation}})
+
+* authenticated negotiation of an application protocol (TLS uses ALPN
+  {{?RFC7301}} for this purpose)
+
+* for the server, assurance that the client can receive packets that are
+  addressed with the transport address that is claimed by the client (see
+  {{source-address-token}})
+
+Details of how TLS is integrated with QUIC is provided in more detail in
+{{QUIC-TLS}}.
+
+
+## Transport Parameters
+
+During connection establishment, both endpoints make authenticated declarations
+of their transport parameters.  These declarations are made unilaterally by each
+endpoint.  Endpoints are required to comply with the restrictions implied by
+these parameters; the description of each parameter includes rules for its
+handling.
+
+The format of the transport parameters is the TransportParameters struct from
+{{figure-transport-parameters}}.  This is described using the presentation
+language from Section 3 of {{!I-D.ietf-tls-tls13}}.
+
+~~~
+   uint32 QuicVersion;
+
+   enum {
+      SFCW(0),
+      CFCW(1),
+      MSPC(2),
+      ICSL(3),
+      TCID(4),
+      COPT(5),
+      (65535)
+   } TransportParameterId;
+
+   struct {
+      TransportParameterId parameter;
+      opaque value<0..2^16-1>;
+   } TransportParameter;
+
+   struct {
+      select (Handshake.msg_type) {
+         case client_hello:
+            QuicVersion negotiated_version;
+            QuicVersion initial_version;
+
+         case encrypted_extensions:
+            QuicVersion supported_versions<2..2^8-4>;
+      };
+      TransportParameter parameters<30..2^16-1>;
+   } TransportParameters;
+~~~
+{: #figure-transport-parameters title="Definition of TransportParameters"}
 
-The transport component of the handshake is responsible for exchanging and
-negotiating the following parameters for a QUIC connection.  Not all parameters
-are negotiated, some are parameters sent in just one direction.  These
-parameters and options are encoded and handed off to the crypto handshake
-protocol to be transmitted to the peer.
+The `extension_data` field of the quic_transport_parameters extension defined in
+{{QUIC-TLS}} contains a TransportParameters value.  TLS encoding rules are
+therefore used to encode the transport parameters.
 
-#### Encoding
+Definitions for each of the defined transport parameters are included in
```

Comment:
I would remove the para above from the transport params PR, and modulo comments below, add the rest of the text into a non-controversial transport params PR.